flocker/apiclient/test/test_client.py

```diff
@@ -31,23 +42,22 @@ def make_clientv1_tests(client_factory, synchronize_state):
     control of this process. So when testing a real client it will be
     talking to a in-process server.
 
-    :param client_factory: 0-argument callable that returns a
-        ``IFlockerAPIV1Client`` provider.
+    The ``TestCase`` should have two 0-argument methods:
```

Comment:
:question: You prefer this style? It forces us to create a subclass, but we do that anyway as a place to hang the TestCase specific documentation. So fine.